testlib/src/commonMain/kotlin/maryk/test/ByteCollector.kt

```diff
@@ -1,6 +1,6 @@
 package maryk.test
 
-/** Collects bytes into a byteArray and enables to read them afterwards*/
+/** Collects bytes into a [ByteArray] and enables to read them afterward. */
```

Comment:
a bit of unrelated tidying, and using an IntelliJ grammar quickfix (personally I think "afterwards" is fine English, but I wanted to get rid of the red squiggly warning)